discard_nth_split: 2
huber_loss_tdoa: 1.0
n_iter: 10
n_splits: 5
robust_loss_fn: 1
scale_res_a2a: 1.0
scale_res_prior: 10
scale_res_tdoa: 1
tag_ranging_observation_variance: 0.005
tag_ranging_synth_variance: 0.1
td: 0.0
topic_pose: ''
topic_range: ''
mode_fine_opt: true